tests/regression_tests/deplete/last_step_reference_materials.xml

```diff
@@ -0,0 +1,1537 @@
+<?xml version='1.0' encoding='utf-8'?>
+<materials>
+  <material depletable="true" id="1" temperature="293.6" volume="0.06674732823364922">
+    <density units="sum" />
+    <nuclide ao="1e-14" name="Xe135" />
```

Comment:
OK. looks like there are redundant isotopes in the file as it stands. Hm. Going to have to look into this. This PR never ends...